### PR TITLE
bitfinex - refactor transfer

### PR DESCRIPTION
**Pull request by pcriadoperez**
Update Manual
Refactor transfer per other PR comments:
- Use common error message format.
- Accept both unified account and account Id from exchange

### CODE REVIEW

File: js/bitfinex.js [hunk 1]
```diff
@@ -379,10 +379,13 @@ module.exports = class bitfinex extends Exchange {
                     'spot': 'exchange',
                     'margin': 'trading',
                     'funding': 'deposit',
-                    'exchange': 'exchange',
-                    'trading': 'trading',
-                    'deposit': 'deposit',
-                    'derivatives': 'trading',
+                    'future': 'exchange',
+                    'contract': 'trading',
```
Comment (pcriadoperez) [inline js/bitfinex.js:383]: Here I wasn't sure what unified account name we wanted to use. But I thought `contract` is what made the most sense.

Comment (kroitor) [reply js/bitfinex.js:383]: In the unified scope we use:
- `spot`
- `margin`
- `future` (for settled futures contracts)
- `swap` (for perpetual futures aka perpetual swaps)
- `option` (for put option contracts and call option contracts)

Comment (kroitor) [reply js/bitfinex.js:383]: If you can, please, add the edits to match the above types.
Related: https://github.com/ccxt/ccxt/issues/11121

File: js/bitfinex.js [hunk 2]
```diff
@@ -379,10 +379,8 @@ module.exports = class bitfinex extends Exchange {
                     'spot': 'exchange',
                     'margin': 'trading',
                     'funding': 'deposit',
-                    'exchange': 'exchange',
-                    'trading': 'trading',
-                    'deposit': 'deposit',
-                    'derivatives': 'trading',
+                    'future': 'exchange',
+                    'swap': 'trading',
```
Comment (kroitor) [inline js/bitfinex.js:383]: Of all derivatives-types, I think bitfinex has just the `swap` type (perpetual futures aka perpetual swaps) and no `future` type (no expiring future contracts)

**Pull request merged**